### PR TITLE
Address reviews on #868

### PR DESCRIPTION
**Pull request by shiveshkhaitan**
This addresses the requested changes on #868.

### CODE REVIEW

File: gazebo_msgs/srv/GetModelProperties.srv
Comment (chapulina) [inline gazebo_msgs/srv/GetModelProperties.srv:1]: A couple of problems here:

* This file is needed for the `ros1_bridge`. But this could be fixed adding the remapping rules [here](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/dashing/gazebo_msgs/gazebo_msgs_mapping_rules.yaml).
* This message has already been released on Dashing, so we'd be breaking API if we changed it. Even if we added new fields side-by-side with the old ones, we'd keep API compatible, but break ABI.

I think the best solution is to copy the message and rename it. Unfortunately anything other than `GetModelProperties` would break the pattern with the other services, but it's either that or leaving the current fields. I'll leave it up to you.

File: gazebo_ros/include/gazebo_ros/gazebo_ros_properties.hpp
Comment (chapulina) [inline gazebo_ros/include/gazebo_ros/gazebo_ros_properties.hpp:56]: Mind documenting the services offered by this plugin?

File: gazebo_ros/src/gazebo_ros_properties.cpp
Comment (chapulina) [inline gazebo_ros/src/gazebo_ros_properties.cpp]: I know this was like this on ROS 1, but we could fix it now :innocent: 

We could also fix the fields below to correctly get information for both axes, in case the joint has 2.

File: gazebo_ros/src/gazebo_ros_properties.cpp
Comment (chapulina) [inline gazebo_ros/src/gazebo_ros_properties.cpp]: There's a [SetCoG](https://bitbucket.org/osrf/gazebo/src/9e18a31cda6328c1d8d4ec08b9d1b6cfcdb93585/gazebo/physics/Inertial.hh#lines-115) which takes position and orientation. Do you know off the top of your head if that function is what we want here, @scpeters ?

**Review by chapulina — CHANGES_REQUESTED** [review]
I pushed some changes on 5293a5a, let me know if anything doesn't look good to you.

File: gazebo_ros/src/gazebo_ros_properties.cpp
Comment (scpeters) [reply gazebo_ros/src/gazebo_ros_properties.cpp]: there's an overload that takes an `ignition::math::Pose3d`:

* https://bitbucket.org/osrf/gazebo/src/2b59492f96fcf121b33f290fd5cf0b70b42f4e22/gazebo/physics/Inertial.hh#lines-126:128

the one at line 115 takes roll, pitch, yaw, but it's easier to construct a Pose with the position xyz and quaternion wxyz

File: gazebo_msgs/srv/GetModelProperties.srv
Comment (shiveshkhaitan) [reply gazebo_msgs/srv/GetModelProperties.srv:1]: Can we change the message for `eloquent`? We will keep the same message for dashing.

File: gazebo_ros/src/gazebo_ros_properties.cpp
Comment (shiveshkhaitan) [reply gazebo_ros/src/gazebo_ros_properties.cpp]: I guess you fixed this in 5293a5a

File: gazebo_msgs/srv/GetModelProperties.srv
Comment (chapulina) [reply gazebo_msgs/srv/GetModelProperties.srv:1]: We still need the bridge to work for Eloquent. But here's what we can do:

* On Eloquent, have the fields side-by-side, accept both, but print warnings if the user is setting the old ones
* We probably don't want to change this for F turtle, since people will still be using the bridge for a long time. So we can keep this deprecated state for a few distros.

What do you think?

File: gazebo_ros/src/gazebo_ros_properties.cpp
Comment (chapulina) [reply gazebo_ros/src/gazebo_ros_properties.cpp]: ha, I did, and forgot to delete the comment :grin:

File: gazebo_msgs/srv/GetModelProperties.srv
Comment (shiveshkhaitan) [reply gazebo_msgs/srv/GetModelProperties.srv:1]: SGTM. I have reverted the changes to message for now aac7178

File: gazebo_ros/src/gazebo_ros_properties.cpp
Comment (chapulina) [reply gazebo_ros/src/gazebo_ros_properties.cpp]: Setting the whole CoG pose on f0e9796

**Review by chapulina — APPROVED** [review]
Thank you @nzlz for starting the migration and @shiveshkhaitan for wrapping it up. LGTM, just waiting on CI.

**Pull request merged**
